pkg/scheduler/priorities.go

```diff
@@ -132,3 +134,92 @@ func (n *NodeLabelPrioritizer) CalculateNodeLabelPriority(pod api.Pod, podLister
 	}
 	return result, nil
 }
+
+// BalancedResourceAllocation is favors nodes with balanced resource usage rate.
```

Comment:
This is my last request, I think-- document sources so people know what they need to read if this code needs to be updated.